src/cli.ts

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+import {
+  DEFAULT_PROJECT_NAMES,
+  DEFAULT_CACHED_REPOSITORIES_DIRECTORY_PATH,
+} from './constants';
+import { main } from './main';
+
+main({
+  argv: process.argv,
+  stdout: process.stdout,
+  stderr: process.stderr,
+  config: {
+    cachedRepositoriesDirectoryPath: DEFAULT_CACHED_REPOSITORIES_DIRECTORY_PATH,
+    defaultProjectNames: DEFAULT_PROJECT_NAMES,
+  },
+}).catch((error) => {
+  console.error(error);
+  process.exitCode = 1;
+});
+
+// vi: ft=typescript
```

Comment:
What does this line mean?

Reply:
Oops, this is a Vim modeline to force it to syntax-highlight this file as a TypeScript file, as for some reason it gets confused. This is a bit of a hack that I forgot I left in, so I can figure out some other way to do this.

Reply:
Removed here: [3b18d0e](https://github.com/MetaMask/module-lint/pull/33/commits/3b18d0e342fb4d952c972537ba4dbed4f332f3bc)